compress/README.md

```diff
@@ -0,0 +1,124 @@
+# Compress a file from Cloud Storage
+
+This pipeline uses gzip command in the default `ubuntu` image to
+allow you to download a file from Cloud Storage, compress it, and
```

Comment:
I think you meant to say that you use the `gzip` command only to compress, and the pipeline will access the contents of the file from GCS, push it through `gzip` and save the compressed file back to GCS.  Maybe something like this might be helpful:

> This pipeline provides the use-case of downloading a file from Cloud Storage, compressing it via `gzip`, and pushing the result to Cloud Storage.  The gzip command is provided as part of the default `ubuntu` image.